### PR TITLE
NO-ISSUE cleaning vm pre build

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ pipeline {
         stage('Init') {
             steps {
                 sh 'make clear-all || true'
+                sh 'docker image prune -a'
```
Comment (filanov) [inline Jenkinsfile:25]: Is there a reason not to do system prune?

Comment (filanov) [reply Jenkinsfile:25]: @YuviGold what do you think?

Comment (YuviGold) [reply Jenkinsfile:25]: I think this isn't a job duty to clean a machine. Especially for the day we would run parallel jobs simultaneously.

Comment (filanov) [reply Jenkinsfile:25]: Do we have some periodical job that can do it?

Comment (oshercc) [reply Jenkinsfile:25]: no

**Pull request merged**